bin/xbps-install/fetch_cb.c

```diff
@@ -145,9 +168,9 @@ stat_display(const struct xbps_fetch_cb_data *xfpd, void *cbdata)
 		(void)xbps_humanize_number(totsize, (int64_t)xfpd->file_size);
 	}
 	if (v_tty)
-		fprintf(stderr, "%s: [%s %d%%] %s ETA: %s\033[K\r",
-		    xfpd->file_name, totsize, percentage,
-		    stat_bps(xfpd, xfer), stat_eta(xfpd, xfer));
+		fprintf(stderr, "%s %s: [%s %d%%] %s ETA: %s\033[K\r",
```

Comment:
Progress is embedded within download line. I've tried adding it as a separate line, but it messes up by package verify message and I don't see a good (a simple) way to address this.

Reply:
I think it's good enough